tool.ts

```diff
@@ -118,15 +125,14 @@ export function findLocalToolVersions(toolName: string, arch?: string) {
    arch = arch || os.arch();
    let toolPath = path.join(_getCacheRoot(), toolName);
 
-//todo: verify .complete
    if (tl.exist(toolPath)) {
         let children: string[] = tl.ls('', [toolPath]);
         children.forEach((child: string) => {
             
             if (isExplicitVersion(child)) {
                 let fullPath = path.join(toolPath, child, arch);
-                if (tl.exist(fullPath)) {
-                    versions.push(semver.clean(child));
+                if (tl.exist(fullPath) && tl.exist(`${fullPath}.complete`)) {
+                    versions.push(child);
```

Comment:
i added the .complete check here, and removed the clean (the cache function already does clean now, so the ones that exist on disk should already be clean)